packages/server/modules/gatekeeper/services/subscriptions.ts

```diff
@@ -267,6 +277,8 @@ export const downscaleWorkspaceSubscriptionFactory =
       case 'starter':
       case 'plus':
       case 'business':
+      case 'team':
+      case 'pro':
```

Comment:
This also has its own logic for figuring out the count of free / paid products we should set the subscription state to. Lets just throw for these guys